src/printer.js

```diff
@@ -1776,6 +1789,31 @@ function genericPrintNoParens(path, options, print) {
     // supported by the pretty-printer.
     case "DeclaredPredicate":
       return concat(["%checks(", path.call(print, "value"), ")"]);
+    case "TSAnyKeyword":
+      return "any";
+    case "TSNumberKeyword":
+      return "number";
+    case "TSObjectKeyword":
+      return "object";
+    case "TSStringKeyword":
+      return "string";
+    case "TSVoidKeyword":
+      return "void";
+    case "TSFunctionType":
+      console.log('TSFunctionType', path.getValue())
+      return options.originalText.slice(util.locStart(path.getValue()), util.locEnd(path.getValue()));
+    case "TSArrayType":
+      return concat([path.call(print, "elementType"), "[]"]);
+    case "TSTypeLiteral": 
+      console.log('TSTypeLiteral', path.getValue())
+      return options.originalText.slice(util.locStart(path.getValue()), util.locEnd(path.getValue()));
+    case "TSPropertySignature":
+      console.log('TSPropertySignature', path.getValue())
+      return "TSPropertySignature"
+    case "TSParameterProperty":
+    case "TSTypeReference":
+      console.log('TSParameterProperty', path.getValue())
+      return options.originalText.slice(util.locStart(path.getValue()), util.locEnd(path.getValue()));
```

Comment:
What you can do is in the default clause to print

```js
/*StartTSNodeName*/originalText.../*EndTSNodeName*/
```

This way anything you don't handle is printed as is but has a comment around so it's easy to spot